WIP: Authorization spec fast-follow (#401)
modelcontextprotocol/modelcontextprotocol: docs/specification/draft/basic/authorization.mdx

clients can determine authorization server endpoints and supported capabilities.

### 2.3.1 Authorization Server Location

MCP servers **MUST** implement OAuth 2.0 Protected Resource Metadata ([RFC9728](https://datatracker.ietf.org/doc/html/rfc9728))
specification to indicate the locations of authorization servers. The Protected Resource Metadata document returned by the MCP server **MUST** include

> the `authorization_servers` field containing at least one authorization server.

Reviewer: Should we recommend that there is only one listed for simplicity sake?

Author: I think that makes sense. I will tweak the verbiage.

the [OAuth 2.0 Protected Resource Metadata](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-resource-metadata-13) documentation for

> guidance on implementation details.
> 
> Protected Resource Metadata documents can define multiple authorization servers. Most MCP server
> implementations will only require a single authorization server entry.
> 
> The responsibility for selecting which authorization server to use lies with the MCP client, following the guidelines specified in
> [RFC9728 Section 7.6 "Authorization Servers"](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-resource-metadata-13#name-authorization-servers).
> 

Reviewer 2: ```suggestion
guidance on implementation details. Implementors should note that Protected Resource Metadata documents can define multiple authorization servers. The responsibility for selecting which authorization server to use lies with the MCP client, following the guidelines specified in
[RFC9728 Section 7.6 "Authorization Servers"](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-resource-metadata-13#name-authorization-servers).

```
